lightningd/lightningd.h

```diff
@@ -343,6 +343,9 @@ struct lightningd {
 	/* Tell openingd/dualopend to accept all, allow sending any. */
 	bool dev_any_channel_type;
 
+	/* Allow changing of shutdown output point even if dangerous */
+	bool dev_allow_shutdown_destination_change;
```

Comment:
not sure if this deserves documentation, but for sure a changelog entry to notify users?

Reply:
dev options never get a change log: they're not a formal part of the API.